adoc/deployment-aws.adoc

```diff
@@ -0,0 +1,87 @@
+== Deployment on Amazon AWS
+
+.Preparation Required
+[NOTE]
+You must have completed <<deployment.preparations>> to proceed.
+
+You will use {tf} to deploy the required master and worker cluster nodes (plus a load
+balancer) to {soc} and then use the `skuba` tool to bootstrap the {kube} cluster on
```

Comment:
Why does this state {soc}? Do we run SOC on top of AWS?

Reply:
Fixed.